packages/console-extension/src/index.ts

```diff
@@ -281,7 +281,12 @@ async function activateConsole(
         for (const name in specs.kernelspecs) {
           const rank = name === specs.default ? 0 : Infinity;
           const spec = specs.kernelspecs[name]!;
-          let kernelIconUrl = spec.resources['logo-64x64'];
+          let kernelIconUrl;
+          try {
+            kernelIconUrl = spec.resources['logo-svg'];
+          } catch (error) {
+            kernelIconUrl = spec.resources['logo-64x64'];
+          }
```

Comment:
Thank you for starting this!

1) It seems that the change would also need to be applied in `helper-extension`, and `notebook-extension`
2) the code needs to be adjusted as `spec.resources['logo-svg']` does not rise; it just returns an undefined, which leads to no fallback icon currently:
![image](https://user-images.githubusercontent.com/5832902/188516829-3766f69a-22ac-4d82-84a9-6bd11eb2d646.png)

I would suggest:

```suggestion
          const kernelIconUrl = spec.resources['logo-svg'] || spec.resources['logo-64x64'];
```

Reply:
Also for reference if someone would like to add an icon for xeus-python (since we are already testing against it), it would go into: https://github.com/jupyter-xeus/xeus-python/tree/master/share/jupyter/kernels/

Reply:
I'll make a PR there too :)